src/components/course/CourseSidebarPrice.jsx

```diff
@@ -9,21 +9,29 @@ import {
   useCoursePriceForUserSubsidy,
 } from './data/hooks';
 import { SubsidyRequestsContext } from '../enterprise-subsidy-requests';
-import { LICENSE_SUBSIDY_TYPE } from './data/constants';
+import { ENTERPRISE_OFFER_SUBSIDY_TYPE, LICENSE_SUBSIDY_TYPE } from './data/constants';
+import { UserSubsidyContext } from '../enterprise-user-subsidy/UserSubsidy';
+import ContactAdmin from '../contact-admin';
 
 export const INCLUDED_IN_SUBSCRIPTION_MESSAGE = 'Included in your subscription';
 export const FREE_WHEN_APPROVED_MESSAGE = 'Free to me\n(when approved)';
+export const COVERED_BY_ENTERPRISE_OFFER_MESSAGE = 'The price of this course could be covered with your organization\'s learning credit balance';
```

Comment:
nit: "... is covered" to eliminate the question/confusion of whether it _could be_ covered or not, despite the strikethrough.

Reply:
Oh, also "learning credit" -> "learner credit"?